contracts/IClaimable.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.12;
+
+/// @title Claimable contract interface
+interface IClaimable {
```

Comment:
why Claimable?

Reply:
I wanted to use a different interface name than `IOwned` (conflicting with Bancor's) or `IOwnable` (conflicting with OZ's) and `Claimable` is how a similar construct was previously named